### PR TITLE
Typography, icons

### PR DESCRIPTION
**Pull request by chrisdavery**
https://nodemcu-smartthings.myshopify.com/?preview_theme_id=161737408827

### CODE REVIEW

File: snippets/icon-cart.liquid [hunk 1]
```diff
@@ -7,4 +7,15 @@
   fill="none"
 >
   <path fill="currentColor" fill-rule="evenodd" d="M17,18C15.89,18 15,18.89 15,20A2,2 0 0,0 17,22A2,2 0 0,0 19,20C19,18.89 18.1,18 17,18M1,2V4H3L6.6,11.59L5.24,14.04C5.09,14.32 5,14.65 5,15A2,2 0 0,0 7,17H19V15H7.42A0.25,0.25 0 0,1 7.17,14.75C7.17,14.7 7.18,14.66 7.2,14.63L8.1,13H15.55C16.3,13 16.96,12.58 17.3,11.97L20.88,5.5C20.95,5.34 21,5.17 21,5A1,1 0 0,0 20,4H5.21L4.27,2M7,18C5.89,18 5,18.89 5,20A2,2 0 0,0 7,22A2,2 0 0,0 9,20C9,18.89 8.1,18 7,18Z"/>
```
Comment (heythisisnate) [inline snippets/icon-cart.liquid:9]: pls remove this commented-out stuff

File: assets/konnected.css [hunk 2]
```diff
@@ -1,3 +1,12 @@
+@font-face {
+  font-family: Inter;
+  src: url('https://cdn.shopify.com/s/files/1/2085/3145/files/Inter-Medium.woff2') format('woff2'),
+      url('https://cdn.shopify.com/s/files/1/2085/3145/files/Inter-Medium.woff') format('woff');
+  font-weight: 500;
+  font-style: normal;
+  font-display: swap;
+}
+
```
Comment (heythisisnate) [inline assets/konnected.css:9]: why do we need to add it again? Isn't this already used for the `body-font` css variable?

**Review by heythisisnate — CHANGES_REQUESTED** [review]
It's looking good.
Some typography updates missing:

1. product titles on collection page
2. top menu font should be slightly smaller and black instead of the gray color

[see Figma](https://www.figma.com/file/CT2tx0eWNLaMweTBSAxgVI/Partner-Pricing)
thanks!

File: assets/konnected.css (hunk 2 above)
Comment (chrisdavery) [reply assets/konnected.css:9]: @heythisisnate can't use 500 font weight of Inter shopify, only regular declared
image

Comment (heythisisnate) [reply assets/konnected.css:9]: We're loading Inter regular twice here in the Shopify theme. Just change the Heading font to Inter Medium and then Shopify will automatically import it. This avoids one extra asset fetch on page load.
Screenshot 2023-10-06 at 8 46 36 AM

File: assets/konnected.css
Comment (heythisisnate) [inline assets/konnected.css:1]: Can you remove the heading `line-height: 1.1em` from lines 55-57. It looks like this is making the headings look too compact. I tried removing it in inspector and I think everything looks better.

**Review by heythisisnate — CHANGES_REQUESTED** [review]

File: assets/konnected.css (hunk 2 above)
Comment (chrisdavery) [reply assets/konnected.css:9]: ok ok tnx

**Review by heythisisnate — APPROVED** [review]

**Pull request merged**